Relays/LowTechRelay.cfg

```diff
@@ -32,10 +32,24 @@ CONTRACT_TYPE
 	// wait for the first relay to unlock
 	REQUIREMENT
 	{
-		name = TechResearched
-		type = TechResearched
+		name = Any
+		type = Any
 
-		part = HighGainAntenna5
+		REQUIREMENT
+		{
+			name = TechResearched
+			type = TechResearched
+
+			part = HighGainAntenna5.v2
```

Comment:
I'm not sure if it supposed to be `HighGainAntenna5.v2` or `HighGainAntenna5_v2`, the part itself is named with `_` (see `GameData\Squad\Parts\Utility\DirectAntennas\HG-5_v2.cfg`), but it the savefile I see only `.` variant. I guess it's equivalent?